deploy/roles/deploy.builder/templates/service-assurance/prometheus/prometheus.yaml

```diff
@@ -4,7 +4,9 @@ metadata:
   name: white
   namespace: sa-telemetry
 spec:
-  version: v2.11.0
+  version: v2.3.2
```

Comment:
Is that a downgrade?

Reply:
Nope, it's just an internal version for Prometheus Operator to work against. It was always a made up version. The actual version is v3.11 in the downstream script. I changed the manifest to use the `tag` parameter to define what the actual version is. Think of this as an API version for the Operator.